__tests__/app.test.js

```diff
@@ -75,3 +75,54 @@ describe('GET /api/articles/:article_id', () => {
     });
     });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('should respons with an updated object where the votes have changed by the number provided in the object', () => {
+        return request(app)
+        .patch("/api/articles/11")
+        .send({inc_vote: -333})
+        .expect(201)
+            .then(({ body }) => {
+                expect(body.updatedArticle).toEqual(
+                    expect.objectContaining({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number)
+                    })
+                )
+            expect(body.updatedArticle.votes).toBe(-333)
+            expect(body.updatedArticle.article_id).toBe(11)
+        })
+    });
+    test('should respond with status 400 & bad request when the object value is not a number ', () => {
+        return request(app)
+        .patch("/api/articles/11")
+        .send({inc_vote: "banana"})
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Bad Request');
+    });
+    });
+    test('should respond with status 400 & bad request when the provided object is empty ', () => {
+        return request(app)
+        .patch("/api/articles/11")
+        .send({})
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Bad Request');
+    });
+    });
+    test('should respons with a status 404 error if an invalid ID is provided', () => {
+         return request(app)
+        .patch("/api/articles/99999")
+        .send({inc_vote: 33})
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Invalid ID');
+    });
+    });
+});
```

Comment:
These tests look good but I'd also throw in a test for if the article id in the endpoint isn't a number as this could throw a different error!